### PR TITLE
issue and pr template updated.

### PR DESCRIPTION
**Pull request by lunaticwhat**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/a_issue_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: '[Issue] '
+labels: 'help wanted, question'
+assignees: ''
+
+---
+
+**Describe the issue**
+<!--- A clear and concise description of what the issue is. -->
+
+**Which build did you use?**
```
Comment (memetrollsXD) [inline .github/ISSUE_TEMPLATE/a_issue_report.md]: Change to `branch` or `commit` maybe?

File: .github/ISSUE_TEMPLATE/a_issue_report.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: Issues
+about: Create an issue if you need any help
+title: '[Issue] '
+labels: 'help wanted, question'
+assignees: ''
+
+---
+
```
Comment (memetrollsXD) [inline .github/ISSUE_TEMPLATE/a_issue_report.md:11]: Add:
**Did you look for other closed issues that have the same problem?**=
So we don't get bonked by duplicate issues

Comment (memetrollsXD) [reply .github/ISSUE_TEMPLATE/a_issue_report.md:11]: https://github.com/stevemao/github-issue-templates/blob/master/no-duplicates/ISSUE_TEMPLATE.md

Something like this on top?

File: .github/ISSUE_TEMPLATE/b_bug_report.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (memetrollsXD) [inline .github/ISSUE_TEMPLATE/b_bug_report.md:1]: Same as `a_issue_report`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
+And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
+## Issues fixed by this PR
+
+<!--- Put the links of issues that fixed by this PR here (if any). -->
```
Comment (memetrollsXD) [inline .github/PULL_REQUEST_TEMPLATE.md]: Fix spelling: `Put the links of issues that may be fixed by this PR here (if any)`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Bug fix
+- [ ] New feature 
+- [ ] Enhancement
+- [ ] Documentation
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
+- [ ] I am responsible for any copyright issues with my code if it occurs in the future.
```
Comment (memetrollsXD) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: - [ ] My pull request is unique and no other pull requests have been opened for these changes
- [ X ] I agree that crepes are better than pancakes

**Review by memetrollsXD — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/a_issue_report.md (hunk 1 above)
Comment (lunaticwhat) [reply .github/ISSUE_TEMPLATE/a_issue_report.md]: yeah i was thinking about this. `branch` might be better

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (lunaticwhat) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: LMFAO

File: .github/ISSUE_TEMPLATE/a_issue_report.md (hunk 2 above)
Comment (lunaticwhat) [reply .github/ISSUE_TEMPLATE/a_issue_report.md:11]: sure

File: .github/ISSUE_TEMPLATE/b_bug_report.md (hunk 3 above)
Comment (lunaticwhat) [reply .github/ISSUE_TEMPLATE/b_bug_report.md:1]: issues are for users for those who seek for help
and bugs are literally for bug reporting ig ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 5 above)
Comment (lunaticwhat) [reply .github/PULL_REQUEST_TEMPLATE.md:22]: lmao

**Comment by lunaticwhat** [comment]
@KingRainbow44 sorry had to mention you we really need this asap 😔

**Pull request merged**